arlo-client/src/components/MultiJurisdictionAudit/_mocks.ts

```diff
@@ -33,4 +41,252 @@ export const auditSettings: {
   },
 }
 
-export default auditSettings
+export const roundMocks: {
+  [key in
+    | 'empty'
+    | 'singleIncomplete'
+    | 'twoIncomplete'
+    | 'singleComplete'
+    | 'needAnother']: IRound[]
+} = {
+  empty: [],
+  singleIncomplete: [
+    {
+      endedAt: '',
```

Comment:
this should be `null` i believe. in general, I try to avoid using empty strings to represent empty values, so you can expect that the backend will return `null` for empty cases unless there's a good reason to do otherwise

Reply:
Oooh good point. Yes that should be `endedAt: string | null`